components/ServiceCard/index.js

```diff
@@ -0,0 +1,38 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import useSWR from "swr";
+
+export default function ServiceCard() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { data, isLoading, error } = useSWR(`/api/services/${id}`);
+
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+
+  if (error) {
+    return;
+  }
+
+  return (
+    <>
+      <h2>Service</h2>
+      <article>
+        <Image
+          src={data.image}
+          width={100}
+          height={100}
+          alt="picture of the service"
```

Comment:
The alt text for images should be more descriptive. Consider dynamic descriptions like .

```suggestion
          alt={picture of ${name}}
```

Reply:
We agree with you in principle, but when we attempt to implement your suggestion, it breaks our code. We would appreciate any advice you could give us. 😊